src/main/resources/alfresco/transforms/0100-basePipelines.json

```diff
@@ -7,27 +7,6 @@
         {"transformerName": "imagemagick"}
       ],
       "supportedSourceAndTargetList": [
-        {"sourceMediaType": "application/pdf",                    "targetMediaType": "image/gif" },
```

Comment:
These two transformers don't need to be separate, and makes using them in other pipelines harder. i.e. you need extra containing pipelines.